module/platform/darwin/SensorFilter/src/SensorFilter.cpp

```diff
@@ -474,11 +492,16 @@ namespace module::platform::darwin {
                         footlanding_Hwf[side]                   = Hwt * Htg;
```

Comment:
I would probably add in something to explain the whole Htg concept since it's a little bit interesting as to how it is supposed to work.

`Htg` does some magic so that the "foot down" position is where the foot would be if it were flat. Therefore as the foot "flattens" it becomes true. So even if your foot hits the ground on the angle it doesn't store the angled position as the "foot down" position, it stores the position that foot would be if/when it is flat on the ground